httpapi/coreapi/coreapi.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 const (
-	documentIDParam      = "document_id"
-	versionIDParam       = "version_id"
+	DocumentIDParam      = "document_id"
```

Comment:
why the renaming?

Reply:
to re-use instead of recreating constants. single source of truth is always better